spec/models/member_spec.rb

```diff
@@ -1,6 +1,39 @@
 require "rails_helper"
 
 RSpec.describe Member do
+  describe "scopes" do
+    describe ".first_insurance_holder" do
+      it "returns the first member that is insured" do
+        create(:member, insured: false, requesting_health_insurance: true)
+        create(:member, insured: true, requesting_health_insurance: false)
+        joel = create(:member, insured: true, requesting_health_insurance: true)
+
+        expect(Member.insured).to eq [joel]
+      end
+    end
+
+    describe ".after" do
+      it "returns the next member _after_ the provided member" do
+        joel = create(:member)
+        jessie = create(:member)
+        christa = create(:member)
+
+        expect(Member.after(joel)).to eq [jessie, christa]
+        expect(Member.after(jessie)).to eq [christa]
+      end
+    end
+
+    describe ".first_member" do
```

Comment:
the way this is named, it feels like it is redundant of `primary_member` - what about naming in `next_member`?

Reply:
What would you suggest?

Reply:
`next_member` would work for me!

Reply:
what do you think?

Reply:
I think I'm going to remove that scope entirely actually. The ROI on its existence isn't clear or beneficial enough to warrant it needing to be there.  `limit(1).first` is just fine in the context of the area of the code that uses it.